nncf/onnx/graph/transformations/commands_creation.py

```diff
@@ -0,0 +1,30 @@
+"""
```

Comment:
Maybe it's better to call this file `commands_factory`?

Reply:
I think we may use the command_factory name only if the `create_bias_correction_command()` method is the factory method.

Reply:
`command_creation.py`?

Reply:
I renamed to `command_creation.py`.